app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.stories.tsx

```diff
@@ -63,3 +65,62 @@ const route = {
 } as const
 
 export const Intraledger = () => <SendBitcoinConfirmationScreen route={route} />
+
+const successActionMessageMock = {
```

Comment:
please move all this test code to the test file.. like with the other components